src/Model/Item.php

```diff
@@ -821,7 +821,9 @@ private static function prepareOriginPost(array $item): array
 	{
 		$item = DI::contentItem()->initializePost($item);
 
-		if (Photo::setPermissionFromBody($item['body'], $item['uid'], $item['contact-id'], $item['allow_cid'], $item['allow_gid'], $item['deny_cid'], $item['deny_gid'])) {
+		if (!isset($item['allow_cid'])) {
+			Logger::warning('Missing permissions from item array', ['callstack' => System::callstack(20)]);
```

Comment:
Nonetheless, I think that this warning is obsolete, since this missing array value will create a warning anyway. And in the logs we can trace it down.

Reply:
PR https://github.com/friendica/friendica/pull/12643 is created.

Reply:
The logger warning was meant to pinpoint the origin of the call with the Callstack, which isn't included with PHP warnings.

Reply:
Luckily this was good traceable even without that, since there aren't that much calls for this functionality. But with the PR this should be cleaned up.

Reply:
`prepareOriginPost` is only called once in `Item::insert` which is called in 40 places between the core and the addons, so the logger call was definitely warranted for someone not familiar with the recent changes.

Reply:
Done.